go-controller/pkg/ovn/policy.go

```diff
@@ -165,17 +165,23 @@ func (oc *Controller) syncNetworkPolicies(networkPolicies []interface{}) error {
 		return fmt.Errorf("cannot find NetworkPolicy Egress ACLs: %v", err)
 	}
 
-	// if the first egress ACL is correct they should all be correct and not need to update
-	if len(egressACLs) > 0 && egressACLs[0].Direction != nbdb.ACLDirectionFromLport {
-		for _, acl := range egressACLs {
-			acl.Direction = nbdb.ACLDirectionFromLport
-			if acl.Options == nil {
-				acl.Options = map[string]string{"apply-after-lb": "true"}
-			} else {
-				acl.Options["apply-after-lb"] = "true"
-			}
+	// the default multicast egress acls were created with the correct direction, update the others
+	updateEgressAcls := []*nbdb.ACL{}
+	for i := range egressACLs {
+		acl := egressACLs[i]
+		if acl.Direction == nbdb.ACLDirectionFromLport {
+			continue
```

Comment:
Thanks for the comments. The multicast traffic does not involves load balancer. But I checked with @hzhou8 and was told "although the multicast traffic is not related to load balancer, it is better to put all the ACLs at the same OVN stage, so that the ACL priorities will apply properly. So, adding apply-after-lb is needed."

Reply:
fixed and repushed @JacobTanenbaum